.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: false
+warnings_are_errors: false
+
+matrix:
```

Comment:
Is the matrix needed? I'd think we could do:

```
language: r
cache: packages
before_install:
  - cd R
```

Reply:
The matrix is so we can add a build and tests for the Python stuff as well. Example taken from `feather` (and simplified greatly). I agree it looks silly with just the R step. :)